packages/website/webpack.config.js

```diff
@@ -24,8 +24,8 @@ module.exports = {
         extensions: ['.js', '.jsx', 'wasm'],
         alias: {
             '@coveord/plasma-react$': path.resolve(__dirname, '../react/dist/cjs/Entry.js'),
-            '@demo-styling': path.resolve(__dirname, 'src/demo-styling'),
-            '@routes': path.resolve(__dirname, 'src/plasma/routes'),
+            '@styles': path.resolve(__dirname, 'src/styles'),
+            '@pages': path.resolve(__dirname, 'src/pages'),

```

Comment:
the `@` is the new convention for JS dependencies but for folders that belong to the project, wouldn't it be better not to use it?

Reply:
You can do the same for all the folders which is at the root of the `src` folder

Reply:
This alias is needed just for this specific line https://github.com/coveo/plasma/blob/2c0b9e70049f39cae4ce8ef72bce522ebd24a536/packages/website/src/building-blocs/useCodeExample.tsx#L25